src/factory/index.ts

```diff
@@ -0,0 +1,48 @@
+import { ChainId } from '@liquality/cryptoassets';
```

Comment:
I think this should be in it's own module (like `client.ts` or `client/index.ts`), not in index. 
This package (`factory`) is also has swapProvider factory.